### ISSUE TITLE
Fix typos on A11y page.

### ISSUE THREAD

**Issue by CorgiDev** [CorgiDev/CorgiDevSite#168]
An attendee pointed out some typos on my A11y page that I missed.

1. "Learning Opprotunities" should be "Learning Opportunities"
2. The text under the "Accessibility Report Tools" heading has "...check the licenses of the repos before making **nay** modifications...", but should be "...check the licenses of the repos before making **any** modifications..."

**Comment by CorgiDev** [comment]
#169 fixes this issue
